docs/content/doc/packages/cran.en-us.md

```diff
@@ -0,0 +1,96 @@
+---
+date: "2023-01-01T00:00:00+00:00"
+title: "CRAN Packages Repository"
+slug: "packages/cran"
+draft: false
+toc: false
+menu:
+  sidebar:
+    parent: "packages"
+    name: "CRAN"
+    weight: 35
+    identifier: "cran"
+---
+
+# CRAN Packages Repository
+
+Publish [CRAN](https://cran.r-project.org/) packages for your user or organization.
+
+**Table of Contents**
+
+{{< toc >}}
+
+## Requirements
+
+To work with the CRAN package registry, you need to install [the R toolset](https://cran.r-project.org/).
```

Comment:
```suggestion
To work with the R package registry, you need to install [R](https://cran.r-project.org/).
```

There is not real "R toolset", it's only that for system that don't ship C compilers by default you also need "R tools" which is basically a bundle of `gcc` and `gfortran` etc.

Hence I'd suggest to just use "R" here as for most packages only the R interpreter is needed anyway.

Reply:
See above